collector.go

```diff
@@ -59,6 +59,24 @@ var (
 		"Atmospheric pressure measurement in mb",
 		varLabels,
 		nil)
+
+  windDesc = prometheus.NewDesc(
+    prefix+"wind_kmh",
+    "Wind strength in kmh",
+    varLabels,
+    nil)
+
+  windangleDesc = prometheus.NewDesc(
+    prefix+"wind_angle",
+    "Wind angle in degrees",
+    varLabels,
+    nil)
+
+  rainDesc = prometheus.NewDesc(
+    prefix+"rain_mm",
```

Comment:
I do not have a rain sensor. What does this actually measure? mm/day? mm/hour? Does it reset to zero at some point?